pkg/decision/bucketer/murmurhashbucketer.go

```diff
@@ -42,20 +42,22 @@ type Bucketer interface {
 // MurmurhashBucketer generates the bucketing value using the mmh3 algorightm
 type MurmurhashBucketer struct {
 	hashSeed uint32
+	logger   logging.OptimizelyLogProducer
 }
 
 // NewMurmurhashBucketer returns a new instance of the murmurhash bucketer
-func NewMurmurhashBucketer(hashSeed uint32) *MurmurhashBucketer {
+func NewMurmurhashBucketer(sdkKey string, hashSeed uint32) *MurmurhashBucketer {
```

Comment:
little weird that NewMurmurhashBucketer takes sdkKey .

Reply:
good catch.  at that level, it should just take a logger. fixed.